src/display-api/display-api.service.ts

```diff
@@ -126,4 +133,155 @@ export class DisplayApiService {
       );
     }
   }
+
+  /**
+   * Fetches all screens.
+   */
+  async fetchScreens(): Promise<ScreenScreenJsonld[]> {
+    try {
+      const config = await this.getAuthenticatedConfig();
+      const screensApi = new ScreensApi(config);
+      const screens: ScreenScreenJsonld[] = [];
+
+      let page = 1;
+      const itemsPerPage = "24";
+      let fetch = true;
+      while (fetch) {
+        const response = await screensApi.getV1Screens(page, itemsPerPage);
+        // TODO, figure out how to avoid any.
+        const data = response.data as any;
```

Comment:
I'm thinking we're closing in on the point in time where we should figure this out.

So, as far as I can tell the API does not describe the shape of its responses, so "any" is the best the generated client can do (some could argue unknown had been better). I can't really see how we can avoid doing a runtime check and then casting - ideally wrapped in a type guard or something like that to make the cast both reusable and somewhat clean.